scripts/qa_crowbarsetup.sh

```diff
@@ -3647,21 +3645,17 @@ function oncontroller_magnum_service_setup
             --property os_distro=fedora-atomic $service_image_name
     fi
 
+    local service_sles_image_name=openstack-magnum-k8s-image
```

Comment:
it is only creating it when the image isn't uploaded. it doesn't look like the post_install recipe is going to be merge ready soon, so I rebased this change meanwhile as we need to start validating what we intend to ship in the product..